AnsiCon/tools/ansicon.ps1

```diff
@@ -0,0 +1,64 @@
+# ANSICON -i produces a Command Processor AutoRun property with these attributes
+#
+#   * at the beginning of the property
+#   * surrounded by parenthesis
+#   * separated by a single & (when there are existing commands)
+#   * without the file extension
+#
+# Example without existing commands:
+# 
+#   (if %ANSICON_VER%==^%ANSICON_VER^% "path\to\ansicon" -p)
+#
+# Example with existing commands:
+# 
+#   (if %ANSICON_VER%==^%ANSICON_VER^% "path\to\ansicon" -p)&<other-commands>
+#
+$pattern = '(?<ansicon>\(if %ANSICON_VER%==\^%ANSICON_VER\^% ".*" -p\))'
+$pattern_greedy = '(?<ansicon>\(if %ANSICON_VER%==\^%ANSICON_VER\^% ".*" -p\)&*)'
+
+function Remove-AnsiconProperty($key) {
+  $autorun = (Get-ItemProperty -Path $key).AutoRun

```

Comment:
You know what, I think this will throw an exception if the reg key path does not exist. I probably ought to do a test-path first.

```powershell
if(-not(Test-Path $key)) {
  return
}
```